dissect/target/volume.py

```diff
@@ -57,6 +57,9 @@ class VolumeSystem:
         serial: Serial number of the volume system, if any.
     """
 
+    __type__: str = None
+    """A short string identifying the type of volume system."""
```

Comment:
Would it be an idea to add a comment (or extend the docstring) with some usecases for using `__type__` instead of `isinstance()`?

Reply:
Added one